<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>541</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QLabel" name="label_state_5">
       <property name="text">
        <string>not at the table</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_state_1">
       <property name="text">
        <string>not at the table</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Приемов пищи:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_eat_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_knifeR_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_knifeL_3">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_knifeR_4">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_knifeL_5">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_knifeR_3">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label_history_4">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_history_3">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_eat_1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Mordred</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_knifeR_2">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Левый нож:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Persival</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_state_4">
       <property name="text">
        <string>not at the table</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_knifeL_4">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_state_3">
       <property name="text">
        <string>not at the table</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Правый нож:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_state_2">
       <property name="text">
        <string>not at the table</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Состояние:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="label_eat_4">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_knifeL_1">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Lancelot</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_knifeL_2">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>№</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_history_1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Историй рассказал:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_history_5">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_eat_3">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Artur</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="label_eat_5">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Galaxad</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_history_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_knifeR_1">
       <property name="text">
        <string>Free</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_43">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Консоль вывода событий:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>481</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/dining_philosophers.jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>171</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>knife 1:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Knife 5:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Knife 4:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Knife 3:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Knife 2:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_image_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/img/img/knife.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_image_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/img/img/knife.jpg</pixmap>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_image_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/img/img/knife.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_image_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/img/img/knife.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_image_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/img/img/knife.jpg</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>129</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Введите кол-во рыц.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
